player/loadfile.c

```diff
@@ -259,33 +259,32 @@ static void print_stream(struct MPContext *mpctx, struct track *t)
     }
     char b[2048] = {0};
 
-    bool tracks_have_lang = false;
+    int max_lang_length = 0;
```

Comment:
```suggestion
    size_t max_lang_length = 0;
```

Reply:
for the language codes longer than 2GB?

Reply:
For sing conversion. strlen returns size_t, no need to sign extend integer for every comparison.

Reply:
With `size_t` you can no longer use `%d`

Reply:
`%zu`

Reply:
Actually it is used for `%-*s` and `FILL()`, you would have to cast it to int anyway

Reply:
Insane idea, do `(int)strlen(...)` this will fix sign compares ;p